landscape/client/__init__.py

```diff
@@ -0,0 +1,11 @@
+import os
+
```

Comment:
Another possibility is to have these values in landscape/constants.py

Reply:
That's a decent thought, but they aren't really constant. They vary based on the env vars.